packages/cli/default/default-dev.yml

```diff
@@ -151,7 +151,7 @@ server:
     - - $require: '@vckit/remote-server?t=function#WebDidDocRouter'
         $args:
           - keyMapping:
-              Ed25519: JsonWebKey2020 # Ed25519VerificationKey2020 | JsonWebKey2020
+              Ed25519: Multikey # Ed25519VerificationKey2020 | JsonWebKey2020 | Multikey
```

Comment:
The same question above

Reply:
Yes, it can be backward-compatible with both V1 and V2 VCDM.